tests/end-to-end/generic/invalid-requirements.phpt

```diff
@@ -20,9 +20,9 @@ Time: %s, Memory: %s
 
 There were 2 PHPUnit test runner warnings:
 
-1) Method PHPUnit\TestFixture\InvalidRequirementsTest::testInvalidVersionConstraint is annotated using an invalid version requirement: Version constraint ~~9.0 is not supported.
+1) Class PHPUnit\TestFixture\InvalidRequirementsTest is annotated using an invalid version requirement: Version constraint ~~9.0 is not supported.
 
-2) Class PHPUnit\TestFixture\InvalidRequirementsTest is annotated using an invalid version requirement: Version constraint ~~9.0 is not supported.
+2) Method PHPUnit\TestFixture\InvalidRequirementsTest::testInvalidVersionConstraint is annotated using an invalid version requirement: Version constraint ~~9.0 is not supported.
```

Comment:
I'm not sure, but I think this changed because the class metadata are now cached before the method metadata as `PHPUnit\Metadata\Api\Requirements::requirementsNotSatisfiedFor()` are called before `PHPUnit\Metadata\Api\DataProvider::providedData()`.